tests/tier2/tc_2044_subscribe_guest_with_activation_key.py

```diff
@@ -67,11 +67,11 @@ def test_run(self):
                 results.setdefault('step2', []).append(False)
 
             logger.info(">>>step3: Register guest with auto-attach enabled by default, no sku in the key")
-            # guest will be auto-attached the best matched pool, vdc has the high priority
+            # guest will be auto-attached the best matched pool, limit has the high priority
             self.system_register_with_ak(self.ssh_guest(), register_config, ak_name)
             output = self.system_sku_consumed(self.ssh_guest())
-            res1 = self.vw_msg_search(output, virtual_limit_pool_id, exp_exist=False)
-            res2 = self.vw_msg_search(output, virtual_vdc_pool_id, exp_exist=True)
+            res1 = self.vw_msg_search(output, virtual_limit_pool_id, exp_exist=True)
+            res2 = self.vw_msg_search(output, virtual_vdc_pool_id, exp_exist=False)
```

Comment:
how about sat6.5? limit sku has the high priority as well?

Reply:
Thanks for the reminder. 
Added test for Sat6.5.1, tested it through hyperv and xen hypervisors, all testing PASS.
In sum, the same behavior for 6.6 and 6.5. : )